app/src/androidTest/java/com/framgia/wsm/test/screen/login/A1RequestOffTest.java

```diff
@@ -0,0 +1,58 @@
+package com.framgia.wsm.test.screen.login;
+
+import android.support.test.espresso.Espresso;
+import android.support.test.espresso.ViewInteraction;
+import android.support.test.espresso.action.ViewActions;
+import android.support.test.espresso.matcher.ViewMatchers;
+import android.test.ActivityInstrumentationTestCase2;
+import com.framgia.wsm.R;
+import com.framgia.wsm.screen.main.MainActivity;
+import cucumber.api.CucumberOptions;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.When;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withClassName;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withParent;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * Created by Duong on 6/21/2017.
+ */
+@CucumberOptions(features = "features/requestoff.feature")
+public class A1RequestOffTest extends ActivityInstrumentationTestCase2<MainActivity> {
+
+    public A1RequestOffTest(MainActivity mainActivity) {
+        super(MainActivity.class);
+    }
+
+    @Given("^I have a Create request off Screen")
+    public void I_have_a_main_screen() {
+        assertNotNull(getActivity());
+    }
+
+    @When("^I click create request$")
+    public void I_click_create_request() {
+        ViewInteraction floatingActionButton =
+                onView(allOf(withClassName(is("com.github.clans.fab.FloatingActionButton")),
+                        withParent(withId(R.id.float_button_create_request)), isDisplayed()));
+        floatingActionButton.perform(click());
+    }
+
+    @When("^I click create request off (\\S+)$")
+    public void I_click_create_request_off() {
+        ViewInteraction floatingActionButton2 =
+                onView(allOf(withClassName(is("com.github.clans.fab.FloatingActionButton")),
+                        withParent(withId(R.id.float_button_create_request)), isDisplayed()));
+        floatingActionButton2.perform(click());
+    }
+
+    @When("^I click next (\\S+)$")
+    public void I_click_login() {
+        Espresso.onView(ViewMatchers.withId(R.id.button_next)).perform(ViewActions.click());
```

Comment:
WARNING: [UnnecessaryFullyQualifiedName] Unnecessary use of fully qualified name 'Espresso.onView' due to existing static import 'android.support.test.espresso.Espresso.onView'
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/imports.html#UnnecessaryFullyQualifiedName